timesketch/frontend/src/components/Sigma/SigmaList.vue

```diff
@@ -15,104 +15,163 @@ limitations under the License.
-              <i class="fas fa-search"></i>
-            </span>
-            Search
-          </button>
+      <b-table-column field="actions" label="Actions" v-slot="props">
+        <router-link
+          :to="{ name: 'Explore', query: { q: props.row.search_query } }">
+          <i class="fas fa-search" aria-hidden="true"
+            title="Search sketch for all events with this tag."></i>
         </router-link>
+
+        <explore-preview style="margin-left: 10px"
+          :searchQuery="props.row.search_query">
+        </explore-preview>
+
+        <span class="icon is-small" style="cursor: pointer" title="Edit Rule"
+          @click="startRuleEdit(props.row)"><i class="fas fa-edit"></i>
+        </span>
+        <span class="icon is-small" style="cursor: pointer" title="Delete Rule"
+          @click="deleteRule(props.row)"><i class="fas fa-trash"></i>
+        </span>
+      </b-table-column>
+      <b-table-column field="title" label="Search Query" v-slot="props">
+        <code>{{ props.row.search_query }}</code>
       </b-table-column>
 
-      <template #detail="props">
-        <b>{{ props['row']['es_query'] }}</b>
 
-        <pre>{{ JSON.stringify(props['row'], null, 2) }}</pre>
-      </template>
     </b-table>
+
+    <div class="container is-fluid">
+      <b-table v-if="sketchTTP.length > 0" :data="sketchTTP">
+        <b-table-column field="search" label="" v-slot="props" width="1em">
+          <router-link
+            :to="{ name: 'Explore', query: generateOpenSearchQuery(props.row.ts_ttp, 'ts_ttp') }">
+            <i class="fas fa-search" aria-hidden="true"
+              title="Search sketch for all events with this tag."></i>
+          </router-link>
+
+        </b-table-column>
+
+        <b-table-column field="tag" label="TTP" v-slot="props" sortable>
+          <b-tag type="is-info is-light">{{ props.row.ts_ttp }} </b-tag>
+        </b-table-column>
+        <b-table-column field="count" label="Events tagged" v-slot="props"
+          sortable numeric>
+          <explore-preview style="margin-left: 10px"
+            :searchQuery="props.row.ts_ttp">
+          </explore-preview>
+        </b-table-column>
+      </b-table>
+      <span v-else>No events have been tagged yet.</span>
+    </div>
   </div>
+
 </template>
 
 <script>
 import ApiClient from '../../utils/RestApiClient'
-
+import ExplorePreview from '../../components/Common/ExplorePreview'
+import { SnackbarProgrammatic as Snackbar } from 'buefy'
+import { SigmaTemplates } from '@/utils/SigmaRuleTemplates'
 export default {
+  components: { ExplorePreview },
   data() {
     return {
       currentPage: 1,
       ascending: false,
       sortColumn: '',
       perPage: 10,
-      isComposed: false,
-      text: `Place your Sigma rule here and press parse`,
+      editingRule: {},
+      showEditModal: false,
+      save_button_text: "Update", // false: update rule. true: create new in DB
```

Comment:
Why no camelCase on `save_button_text`? The comment on this line also doesn't seem to make sense here, remnant of previous code?

Reply:
actually gonna rename the button to `ActionButton` since Update is different action to Save